elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -40,6 +41,12 @@ $LOCAL-VARIABLES: (
     // This is needed if the icon isn't square
     height: pfe-local(icon--Height);
     margin: pfe-local(icon--Margin);
+    // Customize icon color for pfe-icons
+    --pfe-icon--color: #{pfe-local(icon--Color)};
```

Comment:
--pfe-icon--color is deprecated in favor of `--pfe-icon--Color` now (see line 17 of pfe-icon.scss where the new variable is mapped to the old one to maintain backwards compatibility)

Reply:
Awesome! Switched to the new property.